src/Helper/Fields/Field_Coupon.php

```diff
@@ -60,7 +60,7 @@ public function __construct( $field, $entry, Helper_Abstract_Form $gform, Helper
 	 * @since 4.0
 	 */
 	public function html( $value = '', $label = true ) {
-		$value = $this->value();
+		$value = $this->field->get_value_export( $this->entry );
```

Comment:
@jestonihpi how difficult would it be to use this format for the discount code: `Label (Code): -$1.50`

Currently it uses this format: `Label (Code: -$1.50)`, which looks weird.